nb2kg/_version.py

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-version_info = (0, 0, 1, 'dev')
+version_info = (0, 1, 0)
```

Comment:
isn't best practice to leave the 'dev' for the development version, and only remove when publishing a release?

Reply:
@lresende.  I totally agree, but the entire point of this PR is to publish NB2KG to PyPi.  Once in PyPi, I will update the version info to `(0, 2, 0, 'dev')`.